tests/org.jboss.ide.eclipse.as.ui.bot.test/pom.xml

```diff
@@ -172,7 +173,7 @@
 									<outputFileName>jboss-4.0.5.GA.zip</outputFileName>
 									<md5>${jbosstools.test.jboss-as-4.0.md5}</md5>
 									<unpack>true</unpack>
-								</configuration>
+									<skip>${swtbot.test.skip}</skip>								</configuration>
```

Comment:
@robstryker and why add this here? it seems redundant - that's what's effectively there already, inherited from here:
https://github.com/jbosstools/jbosstools-integration-tests/blob/master/tests/pom.xml#L108
Maybe we should really change it there to use skipTestsOrITests and abandon swtbot.test.skip altogether?